pom.xml

```diff
@@ -295,6 +295,8 @@
     <CodeCacheSize>128m</CodeCacheSize>
     <!-- Needed for consistent times -->
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss z</maven.build.timestamp.format>
+    <joni.version>2.1.26</joni.version>
```

Comment:
The latest version is 2.1.43. Why use an old version?

Reply:
sorry, my fault. Now ,it has been updated to the latest version Joni-2.1.43 and jcodings-1.0.57.